lib/components/MenuPopper/MenuPopper.tsx

```diff
@@ -61,9 +61,6 @@ function MenuPopper(props: MenuPopperProps) {
       transition
       disablePortal={disablePortal}
       className='popper-wrapper'
-      nonce={undefined}
-      onResize={undefined}
-      onResizeCapture={undefined}
```

Comment:
Those props aren't appear in MUI Popper API, I don't know why it was added at the beginning
[https://mui.com/material-ui/api/popper/](https://mui.com/material-ui/api/popper/)

Reply:
I assume they were added to avoid some typescript error: [https://github.com/mui/material-ui/issues/35287](https://github.com/mui/material-ui/issues/35287)
Anyway, if it can be removed - need to remove in all the places where we use it, please open a separate ticket for that and bring back those lines meanwhile

Reply:
I'll bring it back, never mind. I used to work with different procedure, if you're working on a file and see issues, you need to fix it if possible, not related to your task.

Reply:
I understand this approach, but I think it's a cleaner way to follow/revert potential issues - when similar things are handled altogether

Reply:
I agree with it in general. But I think that small fixes of typescript warnings, lint issues that can be fixed, makes the code cleaner. There are lot of issues in the code, so clean it in once will be the most Sisyphus work ever :)

Reply:
I don't think we should clean all typescript errors in one step - that's indeed too much, but if we have several Poppers with the same `nonce`, `onResize`, `onResizeCapture` - it's better to remove them once in all the places (and also to make sure they were added only because of typescript)

Reply:
Ok, makes sense, I didn't know that we have few places with the exact same issue

Reply:
btw, with changing `any` to `React.ReactNode` for `content` I totally agree - since this is 100% harmless